server/helpers/genericBookHelpers.js

```diff
@@ -0,0 +1,61 @@
+import Sequelize from 'sequelize';
+import models from '../database/models';
+
+const { Op } = Sequelize;
+const { Book, Author } = models;
+
+const filter = (title, tag, author, keyword) => {
+  const queryParams = {
+    query: {
+      '$Book.title$': { [Op.iLike]: `%${title}%` },
+      '$Book.tag$': { [Op.iLike]: `${tag}` },
+      '$Author.fullname$': { [Op.iLike]: `%${author}%` }
+    },
+    keyword: {
+      [Op.or]: {
+        '$Book.title$': { [Op.iLike]: `%${keyword}%` },
+        '$Book.tag$': { [Op.iLike]: `${keyword}` },
+        '$Author.fullname$': { [Op.iLike]: `%${keyword}%` }
```

Comment:
The query and keyword seems duplicated, I think they can be joined together by using the **any** attribute

Reply:
> No swagger documentation added

The Swagger docs for this route has already been done in a previous PR

Reply:
They are similar, yes, but not identical/duplicated.